rust/src/builders/tx_builder.rs

```diff
@@ -4508,16 +4530,18 @@ mod tests {
+                    &PlutusScriptWitness::from_script(
```

Comment:
Probably it's good if we add a test for the script ref case as well

Reply:
sure will work on that today